qiskit_qec/operators/base_xp_pauli.py

```diff
@@ -258,8 +264,61 @@ def compose(
         front: bool = False,
         inplace: bool = False,
     ) -> "BaseXPPauli":
-        """_summary_"""
-        pass
+        r"""Return the composition of XPPaulis lists
```

Comment:
Why is `num_y` returning the composition of XPPaulis? `num_y` is used in the Pauli clases for changing in the internal phase representation and for displaying. `num_y` needs to be replaced by the appropriate thing.

Reply:
@dsvandet this seems to be an effect of the diff tool. There is indeed a higher `compose` function: https://github.com/qiskit-community/qiskit-qec/blob/8827962c3b1fdf9809812ce03678615ef991a7e4/qiskit_qec/operators/base_xp_pauli.py#L266
`num_y`/its analogue has not been implemented yet:
https://github.com/qiskit-community/qiskit-qec/blob/8827962c3b1fdf9809812ce03678615ef991a7e4/qiskit_qec/operators/base_xp_pauli.py#L178

Reply:
Thanks!